src/actix/api/service_api.rs

```diff
@@ -31,7 +32,31 @@ async fn telemetry(
     process_response(Ok(telemetry_data), timing)
 }
 
+#[derive(Deserialize, Serialize, JsonSchema)]
+pub struct EnableUpdatingParam {
+    pub enable: bool,
+}
+
+#[put("/enable_updating")]
```

Comment:
In REST naming convention it is prefered to use noun + (optional) action for API endpoints. For example:

```
GET /is_updating_enabled -> GET /lock
PUT /enable_updating -> PUT /lock
```

Reply:
done, I use `write_lock` name instead of `lock`